README.md

```diff
@@ -1,336 +1,81 @@
 # Lightning Loop
- 
 Lightning Loop is a non-custodial service offered by
-[Lightning Labs](https://lightning.engineering/) to bridge on-chain and
-off-chain Bitcoin using submarine swaps. This repository is home to the Loop
-client and depends on the Lightning Network daemon
-[lnd](https://github.com/lightningnetwork/lnd). All of lnd’s supported chain
-backends are fully supported when using the Loop client: Neutrino, Bitcoin
-Core, and btcd.
+[Lightning Labs](https://lightning.engineering/) that makes it easy to move
+bitcoin into and out of the Lightning Network.
 
-In the current iteration of the Loop software, two swap types are supported:
-  * off-chain to on-chain, where the Loop client sends funds off-chain in
-  * on-chain to off-chain, where the Loop client sends funds to an on-chain
-    address using an off-chain channel
+## Features
+- Automated channel balancing
+- Privacy-forward non-custodial swaps
+- Opportunistic transaction batching to save on fees
+- Progress monitoring of in-flight swaps
 
-We call off-chain to on-chain swaps, a **Loop Out**.  The service can be used
-in various situations:
+## Use Cases
+- Automate channel balancing with AutoLoop ([Learn more](https://github.com/lightninglabs/loop/blob/master/docs/autoloop.md))
+- Pay or deposit to a Bitcoin address without closing channels
```

Comment:
Paying to an address isn't really supported by this client if this means pay an arbitrary amount